Import scheme does not work with ts-loader v9

```
  ./src/ui.tsx + 9 modules 61.4 KiB [not cacheable] [built] [code generated]
  ./node_modules/css-loader/dist/cjs.js!./src/style.css 18.9 KiB [built] [code generated]

ERROR in /Users/a.ruddick/Documents/github/saxi/src/ui.tsx
1:29-54
[tsl] ERROR in /Users/a.ruddick/Documents/github/saxi/src/ui.tsx(1,30)
      TS2792: Cannot find module '@rehooks/component-size'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
ts-loader-default_734d3b3c730a45e9

ERROR in /Users/a.ruddick/Documents/github/saxi/src/massager.ts
1:30-46
[tsl] ERROR in /Users/a.ruddick/Documents/github/saxi/src/massager.ts(1,31)
      TS2792: Cannot find module 'optimize-paths'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
```


The problem is the `import *` introduced in https://github.com/nornagon/saxi/commit/decf9ab0bfd80e593e89e26a0bbf8b55ebfd57ad.  It can be fixed by switching the import scheme, and adding `"moduleResolution": "node"`

--- a/src/ui.tsx
+++ b/src/ui.tsx
@@ -1,8 +1,8 @@
 import useComponentSize from "@rehooks/component-size";
 import React, { ChangeEvent, Fragment, useContext, useEffect, useLayoutEffect, useMemo, useRef, useState, useReducer } from "react";
 import ReactDOM from "react-dom";
-import * as interpolator from "color-interpolate"
-import * as colormap from "colormap"
+import interpolator from "color-interpolate"
+import colormap from "colormap"
 
 import {flattenSVG} from "flatten-svg";
 import {PaperSize} from "./paper-size";

--- a/tsconfig.web.json
+++ b/tsconfig.web.json
@@ -4,6 +4,7 @@
     "noImplicitAny": true,
     "module": "es6",
     "target": "es6",
+    "moduleResolution": "node",
     "jsx": "react",
     "allowJs": true
   },

